src/server/Adaptive.ReactiveTrader.Server.Pricing/PriceSourceAdapters/FixerAdapter.cs

```diff
@@ -0,0 +1,92 @@
+using Newtonsoft.Json.Linq;
+using Serilog;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Adaptive.ReactiveTrader.Server.Pricing.PriceSourceAdapters
+{
+  /// <summary>
+  /// Scrapes currency pair data from a website called data.fixer.io and returns the currency pairs
+  /// Note https://fixer.io is only free if there are less than a restricted number of API calls per month.
+  /// </summary>
+  public class FixerAdapter : FixerBaseAdapter, IMarketDataAdapter
+  {
+    private const string requestUriString = "https://data.fixer.io/api/latest";
+    private const string sourceName = "Fixer";
+    private const string API_Key = "APIKEY";
```

Comment:
One thing to be aware of here, when we introduce a proper API key is not to check it in.
We should be populating that through config. We don't want people who download the code and run it to use up our request allocation.
And we should provide a suitable fallback (and log), whether through the hardcoded values or whatever if someone tries to run things without a valid API key

Reply:
Agreed.